### PR TITLE
[TASK] xDebug in readme updated

### PR DESCRIPTION
**Pull request by Cyb10101**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,11 +30,17 @@ services:
       - WEB_DOCUMENT_ROOT=/app/public
       - PHP_DISMOD=ioncube
       # @todo - PHP_SENDMAIL_PATH="/home/application/go/bin/mhsendmail --smtp-addr=global-mail:1025"
-      - XDEBUG_REMOTE_HOST=192.168.178.123
-      - XDEBUG_REMOTE_PORT=9000
-      - php.xdebug.idekey=PHPSTORM
-      - php.xdebug.remote_log=/app/xdebug.log
       - php.display_errors=1
+
+      - php.xdebug.cli_color=1
```
Comment (ChrisB9) [inline README.md]: i'd like to have that extracted to a file instead (excluded by .gitignore) - that gives the user an easier way to temporary tweak the container without changing the docker-compose file itself (which results in less false commits)

Comment (Cyb10101) [reply README.md]: Is only a demo in a readme. The user should know if he would rather have it in the Dockerfile, docker-compose.yaml or in the .env file.

Comment (ChrisB9) [reply README.md]: Whatever we write into our readme is kind of our way of saying "that's how you should do it".
Therefore let's recommend the user how to do it 'best'.

In my opinion, it should be in a different file (the webdevops-team even created an overwrite.ini  which is an interesting style) and neither in an .env, the docker-compose or any Dockerfile.

**Comment by Kanti** [comment]
I will close this PR for now as this config is not needed at the moment

**Pull request closed without merging**